internal/testserver/datastore/crdb.go

```diff
@@ -24,15 +25,19 @@ const (
 	enableRangefeeds = `SET CLUSTER SETTING kv.rangefeed.enabled = true;`
 )
 
+// crdbTester is safe for concurrent use by tests.
```

Comment:
I didn't dive deep into why, but ` mage testds:crdbver 25.2.0` kept failing with "conn busy" when I ran in my machine. It works on Github though. The mysteries of the universe..